pkg/model/stack.go

```diff
@@ -777,9 +783,21 @@ func (svcResources *ServiceResources) IsDefaultValue() bool {
 	return svcResources.CPU.Value.IsZero() && svcResources.Memory.Value.IsZero() && svcResources.Storage.Size.Value.IsZero() && svcResources.Storage.Class == ""
 }
 
-func isPathAComposeFile(path string) bool {
+// isFileCompose checks if the path is a compose file
+// if the env var OKTETO_SUPPORT_STACKS_ENABLED is set to true, it will return true for any file no matter the name
+// if the env var is not set, it will return true for files that start with "compose", "docker-compose" or "okteto-compose"
+func isFileCompose(path string) bool {
 	base := filepath.Base(path)
-	return strings.HasPrefix(base, "compose") || strings.HasPrefix(base, "docker-compose") || strings.HasPrefix(base, "okteto-compose")
+	isComposeFileName := strings.HasPrefix(base, "compose") || strings.HasPrefix(base, "docker-compose") || strings.HasPrefix(base, "okteto-compose")
+	if !env.LoadBooleanOrDefault(stackSupportEnabledEnvVar, false) {
```

Comment:
```suggestion
	if !env.LoadBooleanOrDefault(stackSupportEnabledEnvVar, false) {
```
Shouldn't it be without the negation?

Reply:
No, We want to enter in the if when `env.LoadBooleanOrDefault(stackSupportEnabledEnvVar, false)` is `false` as we want to always return that is a compose if the env var is not enabled. That's why it's negated, but reading again the code I think it's a bit complex to understand directly with the function, so I'm setting a variable in order to make it more readable